### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by wangcy2018**
Modify the support for KylinSec 3.3-6c system

### CODE REVIEW

File: packaging/tools/install.sh [hunk 1]
```diff
@@ -105,6 +105,10 @@ elif echo $osinfo | grep -qwi "debian"; then
 elif echo $osinfo | grep -qwi "Kylin"; then
   #  echo "This is Kylin system"
   os_type=1
+elif echo $osinfo | grep -qwi "KylinSec"; then 
+  # echo "This is KylinSec system" 
+  os_type=2 
+  csudo=""
```
Comment (sangshuduo) [inline packaging/tools/install.sh:111]: May I know why csudo be reset here?

**Review by sangshuduo — COMMENTED** [review]
see my comment

**Comment by wangcy2018** [comment]
hello！Because in the current kylinSec system, using csudo will cause the installation to fail, and there is no solution at present

**Comment by sangshuduo** [comment]
> hello！Because in the current kylinSec system, using csudo will cause the installation to fail, and there is no solution at present

OK. Let me merge it first. Once you have better solution, welcome to submit a new PR. Thanks.

**Comment by sangshuduo** [comment]
BTW, Please sign the CLA to make the process continue.

**Comment by wangcy2018** [comment]
Ok，I am very glad to participate in the development of this product.

**Comment by wangcy2018** [comment]
ok，i have finished.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**